homeassistant/components/github/const.py

```diff
@@ -0,0 +1,27 @@
+"""Constants for the GitHub integration."""
+from datetime import timedelta
+from logging import Logger, getLogger
+from typing import NamedTuple
+
+from aiogithubapi import GitHubIssueModel
+
+LOGGER: Logger = getLogger(__package__)
+
+DOMAIN = "github"
+
+# THIS NEED TO CHANGE!!!!
+CLIENT_ID = "ce3981304697fb012542"
+# THIS NEED TO CHANGE!!!!
+
+DEFAULT_REPOSITORIES = ["home-assistant/core"]
+DEFAULT_UPDATE_INTERVAL = timedelta(seconds=300)
```

Comment:
This seems very aggressive? Should we set it default to an hour?

Reply:
Or maybe have it different per type of data being tracked ?

Reply:
This is what's there now.